roles/kernelci-storage/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+# app_user need to go throught /var/www
+- name: Fix /var/www/ permissions
+  file: path=/var/www/
+        state=directory
+        mode=0755
+  tags:
+  - kernelci
+  - storage
+
+- name: Create /var/www/images
+  file: path=/var/www/images
+        state=directory
+        owner={{ app_user }}
+        group={{ web_user }}
+        mode=0750
+  tags:
+  - kernelci
+  - storage
+
+- name: configure storage vhost
+  template: src=kci-storage.conf
+        dest=/etc/nginx/sites-available/kci-storage.conf
```

Comment:
Can you use the value of `kci_storage_fqdn` here as well for the file name?
It would be the same approach for the backend and frontend.

Reply:
ok